ext.gradle

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2017, TeamDev Ltd. All rights reserved.
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+
+/**
+ *  The versions of the libraries used.
+ *
+ *  This file is used wherever the library versions are required.
+ */
+
+final SPINE_VERSION = "0.9.36-SNAPSHOT"
+
+ext {
+    VERSION = SPINE_VERSION
+    BASE_VERSION = "0.9.35-SNAPSHOT"
+    PROTOBUF_VERSION = '3.3.0'
+    SLf4J_VERSION = "1.7.21"
+    PROTOBUF_DEPENDENCY = "com.google.protobuf:protoc:${project.PROTOBUF_VERSION}"
+    MAVEN_REPOSITORY_URL = 'http://maven.teamdev.com/repository/spine'
+    MAVEN_SNAPSHOT_REPOSITORY_URL = 'http://maven.teamdev.com/repository/spine-snapshots'
+
+    GUAVA_VERSION = "20.0"
+
+    IS_WINDOWS = org.gradle.internal.os.OperatingSystem.current().isWindows()
```

Comment:
The idea of `ext.gradle` is to externalize the reusable dependency list, which is being changed on a regular basis. This variable does not belong to `ext.gradle`.